README.md

```diff
@@ -1 +1,189 @@
-# ruby-Morse-code
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make sure you also fix the broken logo image from your README to your documentation looks more professional. I suggest you open this **[template](https://github.com/microverseinc/readme-template)** then you find that missing image and you can download it or you can even add your own logo if you want.